.buildkite/pipeline.yml

```diff
@@ -16,9 +16,9 @@ steps:
       - scripts/upload_re2_artifacts.sh
     env:
       TARGET_ARCH: amd64
+      TARGET_VARIANT: 'glibc-217'
     agents:
       image: family/kibana-ubuntu-2004
-      imageProject: elastic-images-qa
```

Comment:
Won't this be still required? Probably should be changed to `elastic-images-prod`

Reply:
Or is `imageProject: elastic-images-prod` the default?

Reply:
Yes it's the default.